src/transformers/tokenization_blenderbot.py

```diff
@@ -0,0 +1,238 @@
+import json
+import logging
+import os
+from typing import List
+
+import regex as re
+
+from .tokenization_roberta import RobertaTokenizer
+from .tokenization_utils import PreTrainedTokenizer
+
+
+logger = logging.getLogger(__name__)
+
+
+VOCAB_FILES_NAMES = {
+    "vocab_file": "vocab.json",
+    "merges_file": "merges.txt",
+    # "tokenizer_config_file": "tokenizer_config.json",
+}
+CKPT_3B = "facebook/blenderbot-3B"
+
+logger = logging.getLogger(__name__)
+
+
+class BlenderbotTokenizer(RobertaTokenizer):
```

Comment:
Could there be some docs here?

Reply:
The same model as `DistilBertTokenizer` should work here.